### ISSUE TITLE
Add `notMonitored` status for accessibility features that aren't monitored

### ISSUE THREAD

**Issue by Jackstone92** [Jackstone92/A11yoop#16]
Currently, any accessibility features that are not being monitored result in `.disabled` when queried. This is misleading and there should be a corresponding status for this case.
